### PR TITLE
chore: cleaned up CMS to repo sync script

### PR DESCRIPTION
**Pull request by nickytonline**
Thanks for contributing the Netlify plugins directory!

**Are you adding a plugin or updating one?**

- [ ] Adding a plugin
- [ ] Updating a plugin
- [x] Automation

**Have you completed the following?**

- [x] Read and followed the [plugin author guidelines](https://github.com/netlify/plugins/blob/main/docs/guidelines.md).
- [ ] Included all [required fields](https://github.com/netlify/plugins/blob/main/docs/CONTRIBUTING.md#required-fields) in your entry.
- [ ] Tested the plugin [locally](https://docs.netlify.com/cli/get-started/#run-builds-locally) and [on Netlify](https://docs.netlify.com/configure-builds/build-plugins/#install-a-plugin), using the plugin version stated in your entry.

**Test plan**
Please add a link to a successful public deploy log using the stated version of the plugin. Include any other context reviewers might need for testing.

### CODE REVIEW

File: bin/sync_cms_to_plugins_repo.js [hunk 1]
```diff
@@ -11,6 +11,6 @@ console.log('Checking for CMS updates...')
 console.log('Changes to synchronize', changes)
 console.log('Synchronizing changes to plugins repo...')
 const updatedPlugins = updatePlugins(changes, plugins)
-fs.writeFile('site/plugins.json', JSON.stringify(updatedPlugins, null, 2))
+fs.writeFile('site/plugins.json', `${JSON.stringify(updatedPlugins, null, 2)}\n`)
```
Comment (nickytonline) [inline bin/sync_cms_to_plugins_repo.js:14]: Adds a new line that was previously getting removed when doing the CMS to repo sync.

File: bin/sync_cms_to_repo.sh [hunk 2]
```diff
@@ -9,14 +9,17 @@ npx tsx bin/sync_cms_to_plugins_repo.js
 
 echo "Creating PR \"$PR_TITLE\" for branch $BRANCH_NAME"
 
-// This is the only file we want to commit
+# This is the only file we want to commit
```
Comment (nickytonline) [inline bin/sync_cms_to_repo.sh:12]: I had blinders on and couldn't figure out why I was getting errors about a directory. TLDR; JS comments in Bash. 🙃

File: bin/sync_cms_to_repo.sh [hunk 3]
```diff
@@ -7,16 +7,19 @@ git switch $BRANCH_NAME
...
 git add site/plugins.json
 
-// See if we have any changes. We should.
+# See if we have any changes. We should.
 if [[ -n "$(git status --porcelain)" ]]; then
-  git commit -m $PR_TITLE
+  echo "Creating PR \"$PR_TITLE\" for branch $BRANCH_NAME"
```
Comment (nickytonline) [inline bin/sync_cms_to_repo.sh:16]: Brought messaging about PR creation here so it only logs if there are changes.

File: bin/sync_cms_to_repo.sh [hunk 4]
```diff
@@ -7,16 +7,19 @@ git switch $BRANCH_NAME
...
+  git commit -m "$PR_TITLE"
   git push origin $BRANCH_NAME
   # TODO: Uncomment the line below and delete the one below it once we're happy that this is working well in the test environment
-  # gh pr create --title $PR_TITLE --body "This is an automated PR to sync the CMS to the repo" --label "cms_sync" --label "automerge"
-  gh pr create --title $PR_TITLE --body "DO NOT MERGE, TESTING SYNC: This is an automated PR to sync the CMS to the repo" --label "cms_sync"
+  # gh pr create --title "$PR_TITLE" --body "This is an automated PR to sync the CMS to the repo" --label "cms_sync" --label "automerge"
```
Comment (nickytonline) [inline bin/sync_cms_to_repo.sh:20]: As mentioned, after the next set of tests, if this is all good, I'll set the environment to production and use `gh pr create --title "$PR_TITLE" --body "This is an automated PR to sync the CMS to the repo" --label "cms_sync" --label "automerge"` instead.

**Review by sarahetter — APPROVED** [review]
LGTM!

**Review by ericapisani — APPROVED** [review]

**Pull request merged**